### DISCUSSION TITLE
Is it possible to deploy workers on a separate instance and connect to them from my Django app?

### DISCUSSION THREAD

**Discussion by rasulkireev** [django-q2/django-q2#119] [Q&A]
I don't think this is supported, but is there a way to do that? Let me know if more explanation is needed, I might have been a little vague.

**Comment by GDay** [comment]
It should be doable, but you need to deploy your full app there and then connect the database (and broker, if it's not the database as well) to the one from your main app. So, the database (and broker), should be reachable from outside of the main instance (but you should limit this). You don't have to launch the application on the second instance, you can just run the command to start the cluster. 

Hope that helps.

**Reply by rasulkireev** [reply]
So I basically have 2 identical VMs, but one is running `manage.py runserver` and the other `manage.py qcluster`?

**Reply by GDay** [reply]
Exactly! If you want to spread the load, you can even run multiple VMs with `manage.py qcluster`. See here: https://django-q2.readthedocs.io/en/master/cluster.html#multiple-clusters

If you want to have more control over what tasks are done by what worker, you could also define multiple queues: https://django-q2.readthedocs.io/en/master/cluster.html#multiple-queues (this could come in handy if you, for example, have one very resource heavy task that you always want to have ran on a VM with more resources).
